src/frontend/shared-components/src/confirmation-popup/ConfirmationPopup.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import { ConfirmationPopup } from "./ConfirmationPopup";
+
+export default {
+  title: "ConfirmationPopup",
+  component: ConfirmationPopup
+};
+
+const modal = {
+  show: true,
+  handleShow: () => {},
+  handleClose: () => {},
+  title: "Cancel process?",
+  body1: "Your process will end.",
+  body2: "You will be redirected back to the original application."
+};
+const mainButton = { mainLabel: "main label", mainStyling: "normal-blue btn" };
+const confirmButton = {
+  confirmLabel: "Yes, cancel my process please",
```

Comment:
missing period? not a blocker, might be missing context.

Reply:
this is just in the story so not important in terms of actual wording, so going to leave it :)